test/parallel/test-icu-transcode.js

```diff
@@ -1,8 +1,9 @@
 'use strict';
 
-require('../common');
+const common = require('../common');
 const buffer = require('buffer');
 const assert = require('assert');
+const icu = getPunycode();
```

Comment:
IMHO I think it would be better to have the skip up here instead. Also, there is already a `common.hasIntl` that we should be able to re-use:

```js
// ...
const assert = require('assert');

if (!common.hasIntl) {
  common.skip('missing intl... skipping test');
  return;
}

// ...
```